docs/src/scalar_pds.md

```diff
@@ -0,0 +1,146 @@
+# [Tutorial: Solving scalar production-destruction systems with MPRK schemes](@id tutorial-scalar-pds)
+
+Originally, modified Patankar-Runge-Kutta (MPRK) schemes were designed to solve positive and conservative systems of ordinary differential equations.
+The conservation property requires that the system consists of at least two scalar differential equations. 
+Nevertheless, we can also apply the idea of the Patankar-trick to a scalar production-destruction system (PDS)
+
+```math
+u'(t)=p(u(t))-d(u(t)),\quad u(0)=u_0>0
+```
+
+with nonnegative functions ``p`` and ``d``.
+Since conservation is not an issue here, we can apply the Patankar-trick to the destruction term ``d`` to ensure positivity and leave the production term ``p`` unweighted. 
+A first order scheme of this type, based on the forward Euler method, reads
+
+```math
+u^{n+1}= u^n + Δ t p(u^n) - Δ t d(u^n)\frac{u^{n+1}}{u^n}
+```
+and this idea can easily be generalized to higher order explicit Runge-Kutta schemes. 
+
+By closer inspection we realize that this is exactly the approach the MPRK schemes of [PositiveIntegrators.jl](https://github.com/SKopecz/PositiveIntegrators.jl) use to solve non-conservative PDS for which the production matrix is diagonal. 
+Hence, we can use the existing schemes to solve a scalar PDS by regarding the production term as a ``1×1``-matrix and the destruction term as a ``1``-vector.
+
+# [Example 1](@id example-1)
+
+We want to solve
+
+```math
+u' =  u^2 - u,\quad u(0) = 0.95
+```
+
+for ``0≤ t≤ 10``.
+Here,  we can choose ``p(u)=u^2`` as production term and ``d(u)=u`` as destruction term.
+The exact solution of this problem is
+
+```math
+u(t) = \frac{19}{19+e^{t}}.
+```
+
+Next, we show how to solve this scalar PDS in the way discussed above.
+Please note that must use [`PDSProblem`](@ref) to create the problem.
```

Comment:
```suggestion
Please note that you must use [`PDSProblem`](@ref) to create the problem;
[`ConservativePDSProblem`](@ref) does not work since the problem is not conservative.
```